spec/unit/services/add_product_to_basket_spec.rb

```diff
@@ -0,0 +1,25 @@
+require_relative "../../../lib/services/add_product_to_basket"
+require_relative "../../../lib/model/basket"
+require_relative "../../support/shared_examples/models"
+
+RSpec.describe Store::AddProductToBasket do
+  include_examples :models
+
+  let(:quantity) { 2 }
+  let(:product_id) { 1 }
+  let(:new_basket) { Store::Basket.new(product_id: product_id, quantity: quantity) }
```

Comment:
So for example, is this a good solution? or maybe `add_item_to_basket` is better than `add_basket_item`? Or both are no good?

```
let(:add_basket_item) { Store::Basket.new(product_id: product_id, quantity: quantity) }

  before do
    Store::BASKET << add_basket_item
  end
```